addons/mail/static/src/chatter/web/mail_composer_form.js

```diff
@@ -25,21 +25,35 @@ export class MailComposerFormController extends formView.Controller {
 export class MailComposerFormRenderer extends formView.Renderer {
     setup() {
         super.setup();
+        this.orm = useService("orm");
         // Autofocus the visible editor in edition mode.
         this.root = useRef("compiled_view_root");
-        useEffect((isInEdition, root) => {
-            if (root && root.el && isInEdition && this.props.record.data.composition_comment_option !== "reply_all") {
-                const element = root.el.querySelector(".note-editable[contenteditable]");
-                if (element) {
-                    element.focus();
-                    document.dispatchEvent(new Event("selectionchange", {}));
+        useEffect(
+            (isInEdition, root) => {
+                if (
+                    root &&
+                    root.el &&
+                    isInEdition &&
+                    this.props.record.data.composition_comment_option === "reply_all"
+                ) {
+                    const element = root.el.querySelector(".note-editable[contenteditable]");
+                    if (element) {
+                        element.focus();
+                        document.dispatchEvent(new Event("selectionchange", {}));
+                    }
                 }
-            }
-        }, () => [
-            this.props.record.isInEdition,
-            this.root,
-            this.props.record.resId
-        ]);
+            },
+            () => [this.props.record.isInEdition, this.root, this.props.record.resId]
+        );
+
+        const getActiveMailThread = () => {
+            const resIds = JSON.parse(this.props.record.data.res_ids);
+            const thread = this.mailStore.Thread.insert({
+                model: this.props.record.data.model,
+                id: resIds[0],
```

Comment:
Can `resIds` ever be empty ? If so, and if `id: undefined` is fine here, use:
```suggestion
id: resIds.at(0)
```
if `id: undefined` is not fine, return `undefined` and properly protect usage of this function.